### ISSUE TITLE
Options for language switching

### ISSUE THREAD

**Issue by ruayrod** [spheras/messic#124]
Hey guys, could be added one option for language switching from the user or the administrator?.
Thank you.

**Comment by spheras** [comment]
Hi  @ruayrod,

currently messic web client get the language from the navigator. We support english and spanish, and for the next version (hope this week - 1.1.0) we will have catalan too.  Do you really need to change the messic web language to a different language than the web navigator?  Currently it is not supported, but we could study the difficult/beneffit/priority for the next  version of messic. It could be great if you explain us the benefits of this.

Thanks a lot for this, it is a wonderful help for us to see the opinion and needs from other users.

**Comment by ruayrod** [comment]
Uhm... In spanish:

En ocasiones algunos navegadores de móviles, pinchos androids, etc.., no están en la lengua del dispositivo, por ejemplo, este finde vino un amigo, que lleva una "mierd.." de móvil y el navegador nativo lo tenía en inglés, por lo que no pudo usarlo en castellano desde su móvil...
 Si se pudiera fijar esta opción de alguna forma para saltarse ese problema, sería una ventaja. Por defecto dejarlo como está, pero si quieres definirlo para ese usuario en concreto, que se pueda hacer.

Es una sugerencia, que quizás no la utilice nadie, pero en dispositivos que no dan opción es una buena forma de ganar puntos.

Gracias chicos por vuestro trabajo.

**Comment by spheras** [comment]
Ok, entiendo. Me parece interesante. Voy a valorar incluirlo en la siguiente versión de alguna forma. Dejo abierta esta incidencia como mejora para el siguiente milestone.
x cierto, ¿no usas el cliente android (caso de que cuentes con un android) para conectarte al servidor? me ha parecido entender que usabas el navegador del dispositivo.

Gracias!

**Comment by ruayrod** [comment]
En este caso probamos vía navegador del dispositivo. Yo si llevo la app en el móvil y los androids de casa. El problema es que el móvil de mi amigo no tenia espacio para instalar más aplicaciones.. (que también suele pasar) ya te digo que el móvil era una castaña. O si entras con un iPhone para el que no hay app de momento.. ;)

**Comment by spheras** [comment]
sí, lo de iphone es lo que más me duele, pero es que no he trabajado nunca con IOS.  A ver si se acerca algún valiente que se atreva. :D
